Ginger/GingerCoreNET/ActionsLib/General/ActScreenShot.cs

```diff
@@ -166,6 +165,11 @@ public override void PostExecute()
                     }
                 }
             }
+            string filePath = SaveToFileName + "\\" + FileName + ".jpg";
```

Comment:
string filePath = SaveToFileName + "\\" + FileName + ".jpg";
use Path.Combile() method instead of string concatenation

Also in the dictionary object user Key as "ScreenshotFilePath" i.e. somewhat relevant to what we are implementing.

Reply:
Okay, I'll do the needful. 
Also should we provide both the path as well as  the file name, or only the path for now?

Reply:
it should be path + filename